src/VirtualClient/TestResources/Results_KafkaProducer.txt

```diff
@@ -0,0 +1,5 @@
+150295 records sent, 24406.5 records/sec (2.33 MB/sec), 85.4 ms avg latency, 1253.0 ms max latency.
```

Comment:
Just curious why the mb/sec keeps getting higher, is there a warming up we need to do? We need to make sure that the total runtime doesn't change the result

Reply:
This is generated by Kafka load generating tools shipped by kafka.

Reply:
No I understand that, what I'm saying is we need to make sure that if we need a warm up period

Reply:
No, it is not like it always increases. I have added a producer file from one the instance run.
[2024-02-01t1013356098z-producer-numofrec-5000000-recordsize-100-batchsize-8196-async-3.log](https://github.com/microsoft/VirtualClient/files/14235651/2024-02-01t1013356098z-producer-numofrec-5000000-recordsize-100-batchsize-8196-async-3.log)

Also, warmup that I have used in the code is just to make sure that we produce data once and then keep on consuming it. If we try to produce it every time the memory required will be a lot higher.

Reply:
We will not log producer metrics as it will be executed once, then the consumer will keep on consuming the data and keep logging the metrics to kusto.